catalogs/catalog-hive/src/main/java/com/datastrato/gravitino/catalog/hive/HiveCatalogOperations.java

```diff
@@ -537,6 +558,22 @@ public NameIdentifier[] listTables(Namespace namespace) throws NoSuchSchemaExcep
     }
   }
 
+  boolean isHiveTable(Map<String, String> tableParameters) {
+    if (isIcebergTable(tableParameters)) return false;
+    return true;
```

Comment:
Maybe you can directly use `retrun !isIcebergTable(xxx);` instead.

Reply:
Besides, I suggest you change the method name to `isHiveTableParameters` or something similar.

Reply:
why using `isHiveTableParameters`? it's used to check hive table not parrameter.

Reply:
Yeah, I misunderstood it here